packages/deck/frontend/sirius-components-deck/src/toolbar/DeckToolbar.tsx

```diff
@@ -56,24 +65,54 @@ export const Toolbar = ({ editingContextId, representationId }: ToolbarProps) =>
   return (
     <>
       <div className={classes.toolbar}>
-        <IconButton size="small">
-          <FullscreenIcon />
-        </IconButton>
-        <IconButton size="small" data-testid="fit-to-screen">
-          <AspectRatioIcon />
-        </IconButton>
-        <IconButton size="small">
+        {fullscreen ? (
+          <IconButton
+            size="small"
+            color="inherit"
+            aria-label="exit full screen mode"
+            title="Exit full screen mode"
```

Comment:
Prefer MUI's `<Tooltip title="...">...</Tooltip>` instead of native tooltip.
It's what we use on [diagram](https://github.com/eclipse-sirius/sirius-web/blob/master/packages/diagrams/frontend/sirius-components-diagrams/src/renderer/palette/Palette.tsx#L466) and [portals](https://github.com/eclipse-sirius/sirius-web/pull/3040) for exemple so it would be more consistent.